### ISSUE TITLE
semicolon position with same line comments

### ISSUE THREAD

**Issue by dustindowell22** [postcss/sugarss#40]
input:
```
min-height: em(32) // select
```
output:
```
min-height: em(32) /* <select> */;
```

desired output:
```
min-height: em(32); /* <select> */
```

it's not a big deal, but it seems a bit odd to me.

**Comment by ai** [comment]
I am working on it :). Sorry for delay.

**Comment by ai** [comment]
Done 2ded0bf

**Comment by ai** [comment]
Released in 0.1.5.
